VotingApplication/VotingApplication.Web/Scripts/Directives/VotingStrategy.js

```diff
@@ -14,12 +12,12 @@
                 return '';
             }
 
-            return 'routes/' + pollStrategy + 'Vote';
+            return '../Routes/' + pollStrategy + 'Vote';
         }
 
         return {
             replace: true,
-
+            //templateUrl: votingTemplate()
```

Comment:
We'll want to remove this in a future commit. Best to do it on one of the later PRs than this one though